fastapi/README.md

```diff
@@ -10,14 +10,22 @@ _Note: The included Dockerfiles are not ready for production. See the appropriat
 
 1. Have a [Userfront](https://userfront.com/) account
 1. Have docker installed
-1. Have the following set as environment variables:
```

Comment:
TBH I'm not sure why this change was made, this feels like a personal preference.